test/integration/project_test.go

```diff
@@ -15,7 +15,6 @@ import (
 
 // TestProjectIsNamespace verifies that a project is a namespace, and a namespace is a project
 func TestProjectIsNamespace(t *testing.T) {
-	defer testutil.RequireEtcd(t).Terminate(t)
```

Comment:
I'm going to assume that I'm just missing this, but who's starting etcd now?  The `StartTestMaster`?